contrib/mac/juliarc.jl

```diff
@@ -2,8 +2,8 @@
 
 # Set up environment for Julia OSX binary distribution
 let
-    ROOT = abspath(JULIA_HOME,"..")
-    ENV["PATH"]="$JULIA_HOME:$(ENV["PATH"])"
+    ROOT = abspath(Sys.BINDIR,"..")
+    ENV["PATH"]="$Sys.BINDIR:$(ENV["PATH"])"
```

Comment:
The interpolation here will get `Sys` but not `BINDIR`; you'll need to put parentheses around `Sys.BINDIR`.